homeassistant/helpers/entity.py

```diff
@@ -215,6 +215,9 @@ class EntityDescription:
     name: str | None = None
     unit_of_measurement: str | None = None
 
+    action: str | None = None
+    action_value: Any | None = None
+
```

Comment:
This is a base entity-component and should not be modified.

Reply:
I know, I wanted to demonstrate that it might be a good idea to use specific properties for the button entity, for that you need to change the base-component.

Reply:
> I know, I wanted to demonstrate 

Please clarify if this is demo or a actually PR. In case your intention is the first, please close the PR. Thanks 👍

Reply:
Sorry if I demonstrated in the wrong place. The PR will be closed. Thanks.